var/spack/repos/builtin/packages/gmsh/package.py

```diff
@@ -96,6 +100,9 @@ def cmake_args(self):
         if '+oce' in spec:
             env['CASROOT'] = self.spec['oce'].prefix
             options.append('-DENABLE_OCC=ON')
+        elif '+opencascade' in spec:
+            env['CASROOT'] = self.spec['opencascade'].prefix
+            options.append('-DENABLE_OCC=ON')
```

Comment:
Does this imply that `+oce` and `+opencascade` are mutually exclusive options? In that case you can either use a multi-valued variant or a conflicts statement.

Reply:
Yes, oce is actually a repackaged Open CASCADE as a (aka OpenCASCADE Community Edition). I'll add a conflicts statement.